src/D2L.Security.OAuth2/Keys/Default/Data/JwksProvider.cs

```diff
@@ -22,11 +25,12 @@ Uri jwkEndpoint
 			m_jwkEndpoint = jwkEndpoint;
 		}
 
+		[GenerateSync]
 		async Task<JsonWebKeySet> IJwksProvider.RequestJwksAsync() {
 			try {
 				using( HttpResponseMessage response = await m_httpClient.GetAsync( m_jwksEndpoint ).ConfigureAwait( false ) ) {
 					response.EnsureSuccessStatusCode();
-					string jsonResponse = await response.Content.ReadAsStringAsync().ConfigureAwait( false );
+					string jsonResponse = await response.Content.ReadAsStringAsync().ConfigureAwait(false);

```

Comment:
```suggestion
					string jsonResponse = await response.Content.ReadAsStringAsync().ConfigureAwait( false );
```

(nit, mostly to undo the change)